rockcraft.2.dotnet-runtime-9.0-24.10.yaml

```
@@ -5,5 +5,7 @@ platforms:
   amd64:
   arm64:
     build-on: [amd64]
+  ppc64le:
```

```suggestion
  ppc64el:
```

I believe it's `ppc64el`: https://documentation.ubuntu.com/rockcraft/en/latest/reference/rockcraft.yaml/#platforms

---

Hey, so the rocks are built by the https://github.com/canonical/rocks-pipelines/blob/main/src/rocks/Build.py script. And the supported arches there are:

```py
SUPPORTED_ARCHS = ["amd64", "arm", "arm64", "s390x", "ppc64le"]
```

```py
            if not all([arch in SUPPORTED_ARCHS for arch in build_for]):
                msg = (
                    f"{args.rockcraft_yaml}: "
                    f"the provided target platform '{build_for}' is not supported. "
                    f"Supported target architectures are: {SUPPORTED_ARCHS}"
                )
                raise PlatformValidationError(msg)
```

Additionally, if you look at the dotnet-aspnet's rockcraft.yaml, you'll see that we have been building that for `ppc64le` all along.